### PR TITLE
Add TagGroup anatomy illustrations to index pages in docs

### PR DESCRIPTION
**Pull request by reidbarber**
## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 📝 Test Instructions:

Check the index pages for React Spectrum and React Aria.

### CODE REVIEW

**Review by yihuiliao — APPROVED** [review]

File: packages/dev/docs/pages/assets/component-illustrations/TagGroup.svg
Comment (LFDanLu) [inline packages/dev/docs/pages/assets/component-illustrations/TagGroup.svg:1]: Mega nit: noticed that the TagGroup don't line up quite right when seen side to side:
image

My svg manipulation knowledge is pretty small, but it looks like the image portion of the cards is always 153 px so what do you think about making the following style changes to the svg:

```
    padding= calc((152px - 130px) / 2) 10px;
    height= 130px; 
    width= calc(100% - 20px);
    viewBox="0 -20 415 220"
```
and getting rid of the marginBottom?

image

Comment (reidbarber) [reply packages/dev/docs/pages/assets/component-illustrations/TagGroup.svg:1]: Updated, and that looks correct now.

**Review by ktabors — DISMISSED** [review]

**Review by snowystinger — APPROVED** [review]

**Review by yihuiliao — COMMENTED** [review]
Screenshot 2023-05-23 at 9 34 22 AM

it still looks like it isn't quite aligned yet with the other cards

**Comment by reidbarber** [comment]
> it still looks like it isn't quite aligned yet with the other cards

@yihuiliao Good catch, should be fixed now.

**Review by yihuiliao — APPROVED** [review]

**Review by LFDanLu — APPROVED** [review]

**Review by dannify — APPROVED** [review]

**Pull request merged**